test/integration/scheduler/scheduler_test.go

```diff
@@ -51,6 +51,8 @@ import (
 	"k8s.io/kubernetes/test/integration/framework"
 )
 
+const defaultSchedulerPolicyConfigMap string = "scheduler-custom-policy-config"
```

Comment:
nit: I realize this is just a test, but this is a bit long and I think more Kubernetes-standard would be camel-case, so something like schedulerCustomPolicy